packages/e2e-test-utils-playwright/package.json

```diff
@@ -1,7 +1,6 @@
 {
 	"name": "@wordpress/e2e-test-utils-playwright",
-	"version": "0.0.0",
-	"private": true,
+	"version": "1.0.0-prerelease",
```

Comment:
Are you sure the package reached the stage where APIs can be considered v1? The README file is a bit outdated:

https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-test-utils-playwright/README.md#e2e-test-utils

> Experimental End-To-End (E2E) Playwright test utils for WordPress.
>
> This package is still experimental and breaking changes could be introduced in future minor versions (v0.x). Use it at your own risks.
>
> It works properly with the minimum version of Gutenberg 9.2.0 or the minimum version of WordPress 5.6.0.

Maybe this should be published as v0.x for now?

Reply:
> Are you sure the package reached the stage where APIs can be considered v1? The README file is a bit outdated:

Addressed in #44016. Props @talldan.

> Maybe this should be published as v0.x for now?

Suggestion addressed in 3a96d8886f4fe60595466cbeb01244d3efc36ff4.